### ISSUE TITLE
439245292: Solution for emails to SPARC

### ISSUE THREAD

**Issue by KZzizzle** [ITISFoundation/osparc-issues#111]
##### Please leave your contact information
**Name**
Nina Preuss

**Email**
nina.preuss.work@gmail.com

**Can you provide a short description of what you were doing?**
trying to email the right people and not forget anyone

**Can you provide specific and detailed procedure of what went wrong?**
no email distros available

**How would you like the experience to improve?**
Have Tech Leads determine best technical solution. Per Charles Horn: All,

I was using the mail servers for Bluehost for SPARC.science initially but it stopped working when Joost and I set up domain routing using a secure certificate on SPARC.science, which is currently hosted on AWS.

As a workaround, I’m routing the emails from SPARC.science (i.e., info@sparc.science) to a gmail account (info.sparc.science@gmail.com) that is sent to our sparc-science-mail Slack channel.

Moving forward I can assist as needed. Some ideas:
As a workaround, I’m routing the emails from SPARC.science (i.e., info@sparc.science) to a gmail account (info.sparc.science@gmail.com) that is sent to our sparc-science-mail Slack channel.

Per Andre: Hi Nina, all,

I can’t (or at least don’t know how to) set up google groups using our Auckland University google drive. We have for other projects set up domains as GSuite organisation (which I think is likely what Joost means), so that we can create Google groups using our own domain and manage them via the Google Groups interface. I’m not sure how this is done, but can follow up with our IT team if useful.

Otherwise, we also have lots of experience managing email servers via AWS servers – would just need the access to the domain management to point the required records at the corresponding server (I think).

As with Joost, I’m not sure what the costs would be and would depend on the option we go with. But would guess that tagging on email handling to one of our existing AWS servers would be the easiest and cheapest option for us.

Nina User Story: I'd love a range of distros to include a subset or all of the following:

-DRC_PIs
-DRC_Tech_Leads
-DRC_Moderators
-DRC-All_Staff
-DRC-NIH_Program_Team
-DRC_Curation_Team
-DRC_Project_Coordinators
-DRC_Team_Blue
-DRC_CCB
-DRC_MAP_Team
-DRC_SIM_Team
-DRC_DAT_Team
##### Tell us where you found the issue and what you were trying to do.
**Where is the problem? Copy+paste the hyperlink of the page you see this issue.**
email

**I encountered this issue while I was on the following page:**
SPARC Documentation page

**What action were you doing on the Portal when you experienced this issue?**
Other 

**Please specify if you selected "Other" in the previous question.**

**Is this a specific problem with a particular dataset, or a generalized problem that affects all/many?**
General problem

**Would you like us to keep you in the loop as we work to resolve this issue?**
Sure

**Comment by KZzizzle** [comment]
➤ DRC SIMCORE commented:

Assigned this to the tech leads - please delegate as necessary

**Comment by KZzizzle** [comment]
➤ Joost Wagenaar commented:

This needs planning as this requires a process for who is responsible for maintaining these in addition to the technical developement required to set up email servers, and management around that.

**Comment by KZzizzle** [comment]
➤ Abby Paulson commented:

+1 upvote since it would be very useful to the user research process for GazelleCX to have access to an email address @sparc.science dedicated to participant outreach and scheduling.

**Comment by KZzizzle** [comment]
➤ Abby Paulson commented:

From Joost: 
Regarding email servers, what needs to be done is for SPARC to get a subscription for Google GSuite. That should allow custom domain email addresses to be stored using gmail interface. I think it is a completely fine milestone, and we should discuss which core wants to take this as an action item, and subsequently manage this (could be us, could be any of the other cores, could be the NIH).

**Comment by KZzizzle** [comment]
➤ David Nickerson commented:

This would need to be done by someone with access to the sparc.science domain management, right?

**Comment by KZzizzle** [comment]
➤ Abby Paulson commented:

From the CCB Review: 
Leo - questions, who would pay for this? 
Nina - this could be assigned to someone who has to do the research for the technological solutions, and later it will have to be implemented.

**Comment by KZzizzle** [comment]
➤ Abby Paulson commented:

Nina - a lot of things go through email and people get missed in the copying and relay of information. 
Charles - this gives you the ability to log into the email server?
Nina - no, if I email ccb@sparc.science so we make sure that we don't miss people. This is targeted for all the groups as they work together.

**Comment by KZzizzle** [comment]
➤ Charles Horn commented:

I'm still wondering is this is truely needed. Slack works well particularly for cateloging the history of communication and often you can get a more rapid response with usually less typing than email. Also, it seems that many in the development community use tools like Slack on a regular basis. I just don't fully understand the problem and concerned about diverting resources.

**Comment by KZzizzle** [comment]
➤ Leonardo Guercio commented:

Abby Paulson This is a low priority, but I'm concerned it will get stuck in the Needs Planningphase forever unless someone get assigned to do a spike on what is required from a technical perspective here. Any thoughts on who this should be assigned to?

**Comment by KZzizzle** [comment]
➤ Abby Paulson commented:

It would need to be assigned to someone who has the skills and expertise to provide details about how to do this and point the ticket. It seems pretty straightforward in that step one is to get an email server set up. 

The other steps could be captured in dependent tickets later on such as figuring out who is on what list,  what all the lists would be, and any sparc.science email addresses that would be created. 

Leonardo Guercio

**Comment by KZzizzle** [comment]
➤ David Nickerson commented:

It needs to be assigned to someone with access to the domain management being used for sparc.science, or for the asignee to be given that access. The technical solution will depend on how the domain is being managed.

**Comment by KZzizzle** [comment]
➤ Abby Paulson commented:

Adding to the conversation, from Ellen M. Goldstein - CX via email regarding the SPARC Stories, but relevant to the email server solution: 

_Email Server: 
We provided an additional use case for the email server when we updated the Contact Us Wrike ticket, which may be ready to review for the next CCB meeting, so let's try again.

When a user contacts the SPARC portal, and wants us to communicate back with them, the natural and intuitive email response would come from sparc.science email address.
There are both ethical and privacy concerns with members of the DRC using their personal email addresses to communicate with the portal users, as well as it not being viewed as professionally as we would like.

1. Imagine, if you are the Editor of The Journal of Neuroscience Magazine, and you click "Contact Us", and go to a dropdown for "Press Inquiry", that the person who returns the email would not come from SPARC.Science, It may be difficult for the Editor to track the conversation, etc. 
2. If you are from University of Alabama, working on a study and heard of an embargoed experiment and click on Contact Us, and go to a dropdown for "Interested in Collaboration" and then receive an email back from MBF, this also is not a cohesive user experience.
3. And, if GazelleCX is working on behalf of SPARC to collect user stories, I'd like to have ellen@sparc.science address to give confidence in the researcher that we are representing SPARC. It will also save time and cycles...

Having a SPARC.Science email server also demonstrates COHESION and INTEGRATION of the DRC and SPARC overall. _

**Comment by KZzizzle** [comment]
➤ Abby Paulson commented:

As we move this forward, it is likely we will have to do a DAR to determine which email provider to use.

**Comment by KZzizzle** [comment]
➤ Abby Paulson commented:

David Nickerson - this ticket is related with the action item from the last Technical Lead meeting (Andre will follow up to determine how the domain is currently managed and come up with some options for email and mailing lists.)

**Comment by KZzizzle** [comment]
➤ David Nickerson commented:

We now know that sparc.science is, and always has been, managed by Blackfynn. Waiting to hear back from Jeremy Lecoeur as to the specifics before we can move forward.

**Comment by KZzizzle** [comment]
➤ Jeremy Lecoeur commented:

David Nickerson which specific info do you need, I'll make sure to get that to you ASAP

**Comment by KZzizzle** [comment]
➤ David Nickerson commented:

We need to know how the domain is managed, who has access, etc. in order to know what options might be possible for an email solution and how we could go about implementing any particular solution...

**Comment by KZzizzle** [comment]
➤ Jeremy Lecoeur commented:

followers I messaged David Nickerson directly through slack to sync up about the info I had ans what was needed. He said that he will start a DAR process in the next few days

**Comment by KZzizzle** [comment]
➤ Abby Paulson commented:

See email solution tab on the DAR: https://docs.google.com/spreadsheets/d/18d228Sw-THUt96_Liae4wHIUrhFjEPYuLhL-8H8a3xc/edit#gid=1267955070
